### ISSUE TITLE
Make sure we support new regions (e.g. us-east-2)

### ISSUE THREAD

**Issue by justinsb** [kubernetes/kops#1354]
In particular for e2e.  We might need to use the alpha channel in e2e.

cc @zmerlynn

**Comment by hrach** [comment]
👍 `eu-west-2`

**Comment by chrislovecnm** [comment]
Did we get our PR merged into Kubernetes?

**Comment by justinsb** [comment]
I think us-east-2 has been in there for a while ... it's only london we're waiting on.

The blocker here was the image.  If we're using the alpha channel, we get this image.

**Comment by justinsb** [comment]
The latest image in the alpha channel does support the latest regions.  More regions are blocked on upstream k8s support.  Moving to 1.5.1

**Comment by chrislovecnm** [comment]
Closing as we have support
